draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1041,20 +1080,21 @@ pass the following result on to the application for further processing:
 
 <{{examples/complex_ekyc/verified_contents.json}}
 
-## Example 4 - W3C Verifiable Credentials Data Model (work in progress)
+## Example 4a - W3C Verifiable Credentials Data Model v2.0, not using JSON-LD
 
-This example illustrates how the artifacts defined in this specification can be
-represented using W3C Verifiable Credentials Data Model as defined in
-[@VC_DATA].
+This example illustrates how to use the artifacts defined in this specification to secure a payload
+that is represented as a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]
+and does not use JSON-LD. The example uses a content type `credential-claims-set+json` defined in [@VC_JWT], Section 3 in a `cty` JOSE Header value.
 
-SD-JWT is equivalent to an Issuer-signed W3C Verifiable Credential (W3C VC). Disclosures are sent alongside a VC.
+SD-JWT is equivalent to an Issuer-signed W3C Verifiable Credential (W3C VC). Disclosures are sent alongside a W3C VC.
 
-A Presentation with a Holder Binding JWT is equivalent to a Holder-signed W3C Verifiable Presentation (W3C VP).
+A Holder-signed Verifiable Presentation as defined in [@VC_DATA_v2.0] would be equivalent to
+a Combined Format for Presentation with a Holder Binding JWT.
 
-Holder Binding is applied and the Holder Binding JWT is signed using a raw public key passed in a `cnf` Claim in a W3C VC (SD-JWT).
+In this example, Holder Binding is applied and Verifiable Presentation can be signed using
+a raw public key passed in a `cnf` Claim in the SD-JWT.
```

Comment:
not sure we want to call JWK a "raw public key".... I think `cnf` also supports by reference / URL... do we want to support both?

Reply:
```suggestion
a Holder's public key passed in a `cnf` Claim in the SD-JWT.
```